client/packages/invoices/src/Prescriptions/LineEditView/hooks/useDraftPrescriptionLines.tsx

```diff
@@ -54,7 +54,7 @@ export const useDraftPrescriptionLines = (
     const stockLines = uniqBy(
       [...data.nodes, ...invoiceLineStockLines],
       'id'
-   ).filter(stockLine => !stockLine.onHold ); // Filter out on hold stock lines
+    ).filter(stockLine => !stockLine.onHold); // Filter out on hold stock lines
```

Comment:
Nit -  you can remove unnecessary changes in self review to avoid clutter in PR.